makeflow/src/makeflow.c

```diff
@@ -2065,6 +2091,12 @@ if (enforcer && wrapper_umbrella) {
 
 	if(local_queue)
 		batch_queue_delete(local_queue);
+	
```

Comment:
Add comment
"Set the abort and failed flag for batch_job_mesos mode. Since `batch_queue_delete(struct batch_queue *q)` will call `batch_queue_mesos_free(struct batch_queue *q)`, which is defined 
in `batch_job/src/batch_job_mesos.c`. Then this function will check the abort and failed status of 
the batch_queue and inform the makeflow mesos scheduler."